vcs/src/main/java/org/openjdk/skara/vcs/openjdk/Issue.java

```diff
@@ -30,7 +30,7 @@
     private final String shortId;
     private final String description;
 
-    private final static Pattern relaxedIssueParsePattern = Pattern.compile("^((?:[A-Z][A-Z0-9]+-)?[0-9]+)[^\\p{Alnum}]+(\\S.*)$");
+    private final static Pattern relaxedIssueParsePattern = Pattern.compile("^((?:[A-Z][A-Z0-9]+-)?[0-9]+)(?:(?:[^\\p{Alnum}]+\\s)|(?:\\s+))(\\S.*)$");
```

Comment:
I think the `[^\\p{Alnum}]+\\s` pattern mostly subsumes the `\\s+` pattern? I think it can be simplified to:
```suggestion
    private final static Pattern relaxedIssueParsePattern = Pattern.compile("^((?:[A-Z][A-Z0-9]+-)?[0-9]+)[^\\p{Alnum}]*\\s(\\S.*)$");
```
This passes the test locally.

Reply:
Thanks, that certainly is nicer!